server/express.js

```diff
@@ -157,14 +158,23 @@ function makeApp(authAddress, cdapConfig, uiSettings) {
       helmet({
         contentSecurityPolicy: {
           directives: {
-            imgSrc: [`'self' data: ${cspWhiteListUrls}`],
+            imgSrc: [`'self' data: ${cspWhiteListUrls}`, 'www.googletagmanager.com', 'https://www.gstatic.com', 'https://ssl.gstatic.com', 'https://www.google-analytics.com'],
```

Comment:
Why do some domains have a scheme (`https`) but not others?

Reply:
the image source one is recommended that way in the doc on the analytics site